src/store/middlewares/account.test.js

```diff
@@ -40,5 +47,41 @@ describe('Account middleware', () => {
     stubGetAccount.restore();
     stubGetAccountStatus.restore();
   });
+
+  it(`should call transactions API methods on ${actionTypes.metronomeBeat} action if account.balance changes`, () => {
+    store.getState = () => (state);
+    const stubGetAccount = stub(accountApi, 'getAccount').resolves({ balance: 10e8 });
+    const stubTransactions = stub(accountApi, 'transactions').resolves(true);
+
+    middleware(store)(next)({ type: actionTypes.metronomeBeat });
+
+    expect(stubGetAccount).to.have.been.calledWith();
+    // TODO why next expect doesn't work despite it being called according to test coverage?
+    // expect(stubTransactions).to.have.been.calledWith();
+
+    stubGetAccount.restore();
+    stubTransactions.restore();
+  });
+
+  it(`should call store.dispatch(fetchAndUpdateForgedBlocks(...)) on ${actionTypes.metronomeBeat} action if account.balance changes and account.isDelegate`, () => {
+    state.account.isDelegate = true;
+    store.getState = () => (state);
+    const stubGetAccount = stub(accountApi, 'getAccount').resolves({ balance: 10e8 });
+    const stubGetAccountStatus = stub(accountApi, 'getAccountStatus').resolves(true);
+
+    middleware(store)(next)({ type: actionTypes.metronomeBeat });
+
+    expect(stubGetAccount).to.have.been.calledWith();
```

Comment:
This test is not related to title. 
The issue is, Actions that we create using Redux thunk don't return object. They return a function which accepts `dispatch` as argument. but you're using just like other actions.

Other than how you've used it, the actual issue is as described above it best called inside component class.